Source/p_extnodes.c

```diff
@@ -308,9 +302,7 @@ void P_LoadNodes_ZDBSP (int lump, boolean compressed)
 	// release the original data lump
 	W_CacheLumpNum(lump, PU_CACHE);
 	free(zstream);
-#else
 	I_Error("P_LoadNodes: Compressed ZDBSP nodes are not supported!");
```

Comment:
This line need to be removed if we unconditionally build with ZLIb support.

Reply:
Got it.